vm_supervisor/models.py

```diff
@@ -187,16 +190,27 @@ def start_watching_for_updates(self, pubsub: PubSub):
             )
 
     async def watch_for_updates(self, pubsub: PubSub):
-        await pubsub.msubscribe(
-            self.original.code.ref,
-            self.original.runtime.ref,
-            self.original.data.ref if self.original.data else None,
-            *(
-                volume.ref
-                for volume in (self.original.volumes or [])
-                if hasattr(volume, "ref")
-            ),
-        )
+        if self.is_instance:
+            await pubsub.msubscribe(
+                self.original.rootfs.parent,
```

Comment:
I don't think we care about updates to the root file system for instances. We create the instance as a copy of the rootfs at that point in time and then let it live it's life. The user may have written data to their root partition that we should not overwrite.